traefik/templates/_podtemplate.tpl

```diff
@@ -383,6 +383,34 @@
           - "--tracing.addinternals"
           {{- end }}
 
+          {{- with .Values.tracing }}
+          {{- with .sampleRate }}
+          - "--tracing.sampleRate={{ . }}"
+          {{- end }}
+
+          {{- with .serviceName }}
+          - "--tracing.serviceName={{ . }}"
+          {{- end }}
+
+          {{- range $name, $value := .globalAttributes }}
+          - "--tracing.globalAttributes.{{ $name }}={{ $value }}"
+          {{- end }}
+
+          {{- range $index, $value := .capturedRequestHeaders }}
+          - "--tracing.capturedRequestHeaders[{{ $index }}]={{ $value }}"
+          {{- end }}
+
+          {{- range $index, $value := .capturedResponseHeaders }}
+          - "--tracing.capturedResponseHeaders[{{ $index }}]={{ $value }}"
+          {{- end }}
+
+          {{- if gt (len .safeQueryParams) 0 }}
```

Comment:
This has been introduced in 3.1 if I remember correctly. Please add a version check in `requirements.yaml`.